persistence/src/commonMain/db_user/com/wire/kalium/persistence/MessageAssetView.sq

```diff
@@ -45,3 +45,25 @@ AND isEphemeral = FALSE
 ORDER BY date DESC
 LIMIT :limit
 OFFSET :offset;
+
+getAssetMessagesByConversationIdAndMimeTypes:
+SELECT * FROM MessageDetailsView
+WHERE conversationId = :conversationId
+AND visibility IN :visibility
+AND contentType IN :contentTypes
+AND assetMimeType NOT IN :mimeTypes
+AND assetId IS NOT NULL
+AND expireAfterMillis IS NULL
+ORDER BY date DESC
+LIMIT :limit
+OFFSET :offset;
+
+countAssetMessagesByConversationIdAndMimeTypes:
+SELECT COUNT(*) FROM MessageDetailsView
+WHERE conversationId = :conversationId
+AND visibility IN :visibility
+AND contentType IN :contentTypes
+AND assetMimeType NOT IN :mimeTypes
+AND assetId IS NOT NULL
+AND expireAfterMillis IS NULL
+ORDER BY date DESC;
```

Comment:
There was a thought of adding the values (visibility, contentType and assetMimeType) here direclty, BUT in any case those values change we would need to change them manually here as well, so instead of worrying about this in the future we keep it as it is (receiving the values).